back/app/controllers/web_api/v1/phases_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-class WebApi::V1::PhasesController < ApplicationController
-  before_action :set_phase, only: %i[show update destroy survey_results submission_count]
+class WebApi::V1::PhasesController < ::ApplicationController
```

Comment:
This change is not necessary, is it? There is only 1 ApplicationController class.
I see the prefix `::` is also present in back/app/controllers/web_api/v1/projects_controller.rb, but it is not needed there either.